functionary/schema.py

```diff
@@ -135,13 +138,37 @@ def append_new_param_info(
     if comment_info is not None:
         # if depth == 0:  # format: //comment\nparam: type
         info_list.append(f"{offset}{comment_info}")
+        if len(examples_info) > 0:
+            for example in examples_info:
+                info_list.append(f"{offset}{example}")
         info_list.append(f"{offset}{param_declaration}")
     # else:  # format: param: type  // comment
     #     info_list.append(f"{offset}{param_declaration}    {comment_info}")
     else:
         info_list.append(f"{offset}{param_declaration}")
 
 
+def get_examples_info(param_name: str, examples: List) -> List:
+    """get information about examples provided
+
+    Args:
+        param_name (str): _description_
+        examples (List): _description_
+
+    Returns:
+        List: _description_
+    """
+    examples_list = [f"// Example {param_name}:"]
+    for example in examples:
+        if isinstance(example, dict) or isinstance(example, list):
+            example_str = json.dumps(example)
+            examples_list.append(f"// {example_str}")
+        else:
+            examples_list.append(f"// {str(example)}")
```

Comment:
I think should handle if example contains "\n" or not. For example, example="hello\nhi" 
--> would become: 
```
// hello
hi
```

Reply:
Make sure each comment is on one line.